modular_skyrat/code/modules/research/techweb/all_nodes.dm

```diff
@@ -36,3 +36,11 @@
 /datum/techweb_node/botany/New()
 	design_ids += "prisonerbiogenerator"
 	. = ..()
+
+/datum/techweb_node/charge_weaponry
+	id = "charge_weaponry"
+	display_name = "Advanced Pulse-shot Weapons"
+	description = "Utilizing pulse-slung ballistic charge technology, we can produce advanced ballistic weaponry to suit our needs."
+	prereq_ids = list("weaponry", "adv_weaponry", "emp_adv", "electronic_weapons")
```

Comment:
weaponry is not needed other then flavour,
Adv_weaponry needs normal is why.
If its a flavour then then its a-ok, game wise its still the same point locked wise